bokehjs/src/lib/models/tools/actions/zoom_out_tool.ts

```diff
@@ -18,11 +27,17 @@ export class ZoomOutTool extends ZoomBaseTool {
   static init_ZoomOutTool(): void {
     this.prototype.default_view = ZoomOutToolView
 
+    this.define<ZoomOutTool.Props>(({Boolean}) => ({
+      maintain_focus: [ Boolean, true ],
+    }))
+
     this.register_alias("zoom_out", () => new ZoomOutTool({dimensions: "both"}))
     this.register_alias("xzoom_out", () => new ZoomOutTool({dimensions: "width"}))
     this.register_alias("yzoom_out", () => new ZoomOutTool({dimensions: "height"}))
   }
 
+  get _maintain_focus(): boolean { return this.properties.maintain_focus.get_value() }
```

Comment:
The `.get_value()` should not be needed

Reply:
Without the .get_value() I get `Type 'Property<boolean>' is not assignable to type 'boolean'.`.

Reply:
@mattpap can you comment? I don't see why this should be required, there's almost nowhere in the codebase that we (have to) use `get_value`?

Reply:
It should be `this.maintain_focus`, which is an alias for `this.properties.maintain_focus.get_value()`.

Reply:
Ah, looking to fast, I overlooked the `.properties` in there

Reply:
Removed this ugly getter entirely thanks to @mattpap's suggestion. Looks good from my end now.